templates/energyplus/templates/ref-sys.pxt

```diff
@@ -121,3 +126,122 @@ Refrigeration:Condenser:AirCooled,
     22.2;                    !- Maximum Value of x
 <% end %>
 
+#Afshin Faramarzi and Behzad Rizi, 11/11/2024 - added EnergyManagementSystem to the model to control saturated condensing temperature 
+!-   ===========  ALL OBJECTS IN CLASS: ENERGYMANAGEMENTSYSTEM:ACTUATOR ===========
+
```

Comment:
We need this new block of code to be turned off by default and so that the base case still works as expected. This might require a new parameter and an IF-statement. Something like this:

    parameter "cond_floating", :default=>false, :domain => Boolean # (true | false)
    <% if (cond_floating) %>
    ...
    <% end %>